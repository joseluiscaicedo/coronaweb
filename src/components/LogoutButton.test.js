import React from 'react';
import '@testing-library/jest-dom/extend-expect'
import { render } from '@testing-library/react'
import LogoutButton from './LogoutButton';


describe('validate LogoutButton Component', () =>{
  test('renders content', () => {
    const component = render(<LogoutButton />)

    expect(component.length)>0;

  });
  test('Generates Snapshot for LogoutButton Component', () => {
    const { container } = render(
      <LogoutButton />
    );
    expect(container.firstChild).toMatchSnapshot();
  });

})