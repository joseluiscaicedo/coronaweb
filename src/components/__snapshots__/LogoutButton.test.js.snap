// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validate LogoutButton Component Generates Snapshot for LogoutButton Component 1`] = `
<button>
  Logout
</button>
`;
